docs/quickstart/custom_dataset.md

```diff
@@ -266,6 +267,44 @@ ns-process-data record3d --data {data directory} --output-dir {output directory}
 ns-train nerfacto --data {output directory}
 ```
 
+(spectacularai=)
```

Comment:
I think changing this to `(spectacularai)=` might fix the sphinx error "/home/runner/work/nerfstudio/nerfstudio/docs/quickstart/custom_dataset.md:22: WARNING: 'myst' reference target not found: spectacularai"